extensions/stackdriver/common/constants.h

```diff
@@ -29,6 +29,23 @@ constexpr char kClientResponseBytesMeasure[] = "client/response_bytes_measure";
 constexpr char kClientRoundtripLatenciesMeasure[] =
     "client/roundtrip_latencies_measure";
 
+constexpr char kServerConnectionsOpenCountMeasure[] =
+    "server/connection_open_count_measure";
+constexpr char kServerConnectionsCloseCountMeasure[] =
+    "server/connection_close_count_measure";
```

Comment:
Are all these existing names used in mixer v1?

Reply:
For the size one's yes, but mixer v1 didn't have the connection count metrics, so name is same as here: https://istio.io/docs/reference/config/policy-and-telemetry/metrics/
@douglas-reid is helping update the schema here..

Reply:
Thanks.. will change default to off for the metrics to be reported...

Reply:
Fixed